jira.go

```diff
@@ -196,6 +212,8 @@ func openJiraTicket(flags flags, projectInfo jsn.Json, vulnForJira interface{},
 
 		return responseData, ticketFile, nil, jiraApiUrl
 	}
+	message := fmt.Sprintf("*** ERROR *** Failed to create ticket, vuln ID is empty\n")
```

Comment:
why would `vulnId` be empty, what can cause this scenario?

Reply:
I am not sure it's possible in the real world but if we can't extract the info from the JSON or it doesn't exist for some reason. It's a safety thing, will move the check to earlier to return faster